src/fcmp_pp/fcmp_pp_rust/fcmp++.h

```diff
@@ -219,6 +221,10 @@ CResult output_blinds_new(const uint8_t *o_blind,
 CResult helios_branch_blind(void);
 CResult selene_branch_blind(void);
 
+int new_helios_branch_blind(struct HeliosBranchBlindUnsafe **branch_blind_out);
+
+void free_helios_branch_blind(struct HeliosBranchBlindUnsafe *helios_branch_blind);
```

Comment:
I would rename this to `destroy_helios_branch_blind` to signify that's it's not just freeing memory, but performing actions on the Rust side to cleanup this object. Basically mirroring the difference between "free" and "delete" in C++. Out of the last 3, this is the naming change I feel most strongly about.